lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -177,6 +177,10 @@ def _get_apn_correction_map_list(service_mconfig):
         return service_mconfig.apn_correction_map_list
     return get_service_config_value("mme", "apn_correction_map_list", None)
 
+def _get_federated_mode_map(service_mconfig):
+    if service_mconfig.federated_mode_map.enabled and \
```

Comment:
where is the `federated_mode_map` being pulled from? do we already have this on the mconfig file?

Reply:
my bad, this is already generated on `mconfigs.proto`

Reply:
yep, per our discussion is in lte `mconfig.proto`

```
    FederatedModeMap federated_mode_map = 34;
}
message FederatedModeMap {
    bool enabled = 1;
    repeated ModeMapItem mapping = 2;
}

```